sensors/listener.go

```diff
@@ -220,6 +220,21 @@ func (sensorCtx *SensorContext) listenEvents(ctx context.Context) error {
 				})
 				if err != nil {
 					triggerLogger.Warnf("failed to trigger actions, %v", err)
+					if trigger.DlqTrigger != nil {
+						dlqRetryStrategy := trigger.DlqTrigger.RetryStrategy
+						if dlqRetryStrategy == nil {
+							dlqRetryStrategy = &apicommon.Backoff{Steps: 1}
+						}
+
+						triggerLogger.Warnf("invoking dlqTrigger, %v", err)
+						dlqErr := common.DoWithRetry(dlqRetryStrategy, func() error {
+							return sensorCtx.triggerActions(ctx, sensor, events, *trigger.DlqTrigger)
+						})
+
+						if dlqErr != nil {
+							triggerLogger.Errorf("failed to trigger dlqTrigger, %v for error %v", err, dlqErr)
```

Comment:
Why is `err` logged again here? Would it be enough to log `dlqErr` only?

Reply:
done!